unstructured/partition/pdf_image/pdfminer_processing.py

```diff
@@ -129,3 +87,42 @@ def get_regions_by_pdfminer(
         layouts.append(layout)
 
     return layouts
+
+
+def merge_inferred_layout_with_extracted_layout(
```

Comment:
Would the shorter `merge_inferred_with_extracted_layout()` still communicate? 43 chars is very long for an identifier and starts to produce layout problems on the call.